WooCommerce/Classes/ViewRelated/Orders/Order Creation/OrderForm.swift

```diff
@@ -205,6 +219,58 @@ struct OrderForm: View {
         .notice($viewModel.autodismissableNotice)
         .notice($viewModel.fixedNotice, autoDismiss: false)
     }
+
+    @ViewBuilder private var storedTaxRateBottomSheetContent: some View {
+        VStack (alignment: .leading) {
+            // your sheet content here
+            Text(Localization.storedTaxRateBottomSheetTitle)
+                .bodyStyle()
+                .padding(.top, Layout.storedTaxRateBottomSheetTopSpace)
+                .padding([.leading, .trailing, .bottom])
+                .frame(maxWidth: .infinity, alignment: .leading)
+
+            if let taxRateViewModel = viewModel.storedTaxRateViewModel {
+                TaxRateRow(viewModel: taxRateViewModel, onSelect: nil)
+                    .overlay(
+                        RoundedRectangle(cornerRadius: Layout.storedTaxRateBottomSheetRowCornerRadius)
+                            .stroke(Color(.separator), lineWidth: 1)
+                    )
+                    .padding()
+            }
+
+            Button {
+            } label: {
+                Label {
+                    Text(Localization.storedTaxRateBottomSheetNewTaxRateButtonTitle)
+                        .bodyStyle()
+                } icon: {
+                    Image(systemName: "pencil")
+                        .resizable()
+                        .frame(width: Layout.storedTaxRateBottomSheetButtonIconSize * scale,
+                               height: Layout.storedTaxRateBottomSheetButtonIconSize * scale)
+                        .foregroundColor(Color(.secondaryLabel))
+                }
+            }
+            .padding()
+
+            Button {
+            } label: {
+                Label {
+                    Text(Localization.storedTaxRateBottomSheetClearTaxRateButtonTitle)
+                } icon: {
+                    Image(systemName: "x.circle")
```

Comment:
```suggestion
                    Image(systemName: "xmark.circle")
```

<img width="283" alt="xmark.circle SFSymbol – a multiplication mark in a circle" src="https://github.com/woocommerce/woocommerce-ios/assets/2472348/7853eb64-5381-4ad7-98a4-bfbec9e219bc">

vs

<img width="296" alt="x.circle SF symbol – a letter X" src="https://github.com/woocommerce/woocommerce-ios/assets/2472348/14947819-bd2f-4268-9ddc-7198333d4843">

Reply:
Good hint, done in e8393a38aeeb8c4ebc0c44d44fbdeae91aae0c06